omc3/scripts/betabeatsrc_output_converter.py

```diff
@@ -0,0 +1,448 @@
+"""
+BetaBeat.src Output Converter
+-------------
+
+Script to convert most important output files produced by ``BetaBeat.src`` / ``GetLLM`` into the standard
+format used in ``omc3`` to allow straight forward comparison of the two.
+"""
+from collections import OrderedDict
+from datetime import datetime
+from pathlib import Path
+from typing import Union
+
+import tfs
+from generic_parser.entrypoint_parser import (
+    EntryPointParameters,
+    entrypoint,
+    save_options_to_config,
+    DotDict,
+)
+
+from omc3.definitions import formats
+from omc3.definitions.constants import PLANES
+from omc3.optics_measurements.constants import (AMPLITUDE, AMP_BETA_NAME,BETA_NAME, DELTA, DISPERSION_NAME,
+                                                ERR, EXT, IMAG, MDL, NORM_DISP_NAME, ORBIT_NAME, PHASE,
+                                                PHASE_NAME, REAL, TOTAL_PHASE_NAME)
+from omc3.optics_measurements.toolbox import df_ang_diff, df_diff, df_err_sum, df_ratio, df_rel_diff
+from omc3.utils import contexts, iotools, logging_tools
+
+LOGGER = logging_tools.get_logger(__name__)
+OLD_EXT: str = ".out"
+DEFAULT_CONFIG_FILENAME: str = "old_measurement_converter_{time:s}.ini"
+
+
+def converter_params():
+    params = EntryPointParameters()
+    params.add_parameter(
+        name="inputdir",
+        required=True,
+        type=str,
+        help="Directory with BetaBeat.src output files."
+    )
+    params.add_parameter(
+        name="outputdir", required=True, type=str, help="Output directory for converted files."
+    )
+    params.add_parameter(
+        name="suffix",
+        type=str,
+        default="_free2",
```

Comment:
I would say that the default should be _free since this is what we normally used.

Reply:
Good to know then, I can make that quick change.